kong/runloop/balancer/init.lua

```diff
@@ -466,11 +466,17 @@ local function set_host_header(balancer_data, upstream_scheme, upstream_host, is
   return true
 end
 
-
+local function log(balancer_data, ctx)
```

Comment:
Personally I don't like this function being called `log`, it seems ambiguous to what it is actually doing.